README.md

```diff
@@ -102,7 +117,13 @@ Using the object literal notation, Define an object called student and give it t
 
 Your Answer:
 ```js
-// write code here
+student = {
+  name: Mohamed,
+  age: 27,
+  sayHello: function(){
+    alert('Hi, my name is '+student.name)
```

Comment:
:+1:  One thing to keep in mind is how we can use the benefits of context to access the `name` property of the `student` attribute without referring to the global objects variable by name